src/pages/tBTC/Bridge/Minting/ProvideData.tsx

```diff
@@ -179,17 +196,44 @@ export const ProvideDataComponent: FC<{
         bitcoinNetwork={threshold.tbtc.bitcoinNetwork}
         onSubmitForm={onSubmit}
       />
+      <Checkbox defaultChecked onChange={handleDepositReceiptAgreementChange}>
+        Download Deposit Receipt&nbsp;
+        <Text as="span" color="#888">
+          (Recommended)
+        </Text>
+        <TooltipIcon
+          color="#888"
+          label="
+            The file is extremely important in case you need to make a fast 
+            recovery. This file is important in the rare case of fund recovery.
+            Keep it until you have received your tBTC token. One deposit, one 
+            receipt. This file contains a BTC recovery address, a wallet public 
+            key, a refund public key and a refund lock time of your deposit.
+          "
+          ml={3}
```

Comment:
Let's remove the `ml` property from `TooltipIcon` and add `mr` to recommended `Text` above. The reason for that is the `PopoverContent` is slighty off from the icon right now:

<img width="250" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/40306834/19ea0ce5-e994-4d10-b821-48222bb0fc1f">

Reply:
Ref commit: e4955c23